ee/modules/600-flant-integration/openapi/values.yaml

```diff
@@ -95,3 +95,9 @@ properties:
         x-examples: ["kasakdhaksdhkadkadh"]
         description: |
           Client license key.
+      madisonAuthKey:
```

Comment:
How to change madison auth key if I want, for example, to copy it from my previously deployed cluster?

Reply:
Add generated key storage place into config values docs.

Reply:
<img width="704" src="https://user-images.githubusercontent.com/1055474/189855556-7b8cf8e8-8dba-485a-889a-ac20737e7611.png">